nemo/utils/preemption_callback.py

```diff
@@ -0,0 +1,66 @@
+import signal
```

Comment:
Also, let's change this to nemo/utils/callbacks/preemption.py

Reply:
Yes this has been renamed to preemption.py and moved to nemo/utils/callbacks folder.